wcfsetup/install/files/acp/update_com.woltlab.wcf_5.2.10_orphanedComments.php

```diff
@@ -0,0 +1,29 @@
+<?php
+namespace wcf\acp;
+
+use wcf\data\object\type\ObjectTypeCache;
+use wcf\system\comment\CommentHandler;
+use wcf\system\WCF;
+
+// Earlier versions did remove moderation queues without taking care of
+// comments associated with them.
+$commentObjectTypeID = ObjectTypeCache::getInstance()->getObjectTypeIDByName(
+	"com.woltlab.wcf.comment.commentableContent",
+	"com.woltlab.wcf.moderation.queue"
+);
+
+$sql = "SELECT  objectID
```

Comment:
```suggestion
$sql = "SELECT  DISTINCT objectID
```
This makes the result set smaller in case of multiple comments for a single queue entry.